public/src/game.js

```diff
@@ -0,0 +1,46 @@
+var Game = function(frame) {
+  this.throwList = [];
```

Comment:
Not sure why the Game keeps a complete throwList as it's not needed by the runningTotal